File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -11,16 +11,16 @@ keywords: Assemblies
 
 Steps to convert HTML to PDF in Blazor application
```

Comment:
Add heading tag for this

Reply:
Add description as "The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF document in Blazor.

Reply:
Added description